packages/@okta/vuepress-site/docs/guides/configure-user-scoped-account-management/main/index.md

```diff
@@ -0,0 +1,258 @@
+
+```bash
+{
+  "profile": {
+      "email" : "some.primary.email@okta.com"
+  },
+   "role": "SECONDARY",
+   "sendEmail": true
+}
+```
+
+#### Response body
+
+The requested [My Email object](#my-email-object)
+
+#### Error Responses
+
+If an invalid email is passed to `profile` in the request body, the response returns a 400 BAD REQUEST with error code E0000001.
+
+If the email operation isn't enabled for the request `role` on the org, the response returns a 403 FORBIDDEN with error code E0000038.
+
+If the email already exists for the current user, the response returns a 409 CONFLICT with error code E0000157.
+
+##### Request
+
+```bash
+curl -v -X POST \
+-H "Authorization: Bearer ${token}" \
+"https://${yourOktaDomain}/idp/myaccount/emails"
+```
+
+##### Response
+
+```json
+{
+    "id": "5a8de6071e1b94e0f4ec664b9e4869e8",
+    "status": "UNVERIFIED",
+    "profile": {
+        "email": "secondary.email@okta.com"
+    },
+    "roles": [
+        "SECONDARY"
+    ],
+    "_links": {
+        "self": {
+            "href": "https://${yourOktaDomain}/idp/myaccount/emails/5a8de6071e1b94e0f4ec664b9e4869e8",
+            "hints": {
+                "allow": [
+                    "GET"
+                ]
+            }
+        },
+        "challenge": {
+            "href": "https://${yourOktaDomain}/idp/myaccount/emails/5a8de6071e1b94e0f4ec664b9e4869e8/challenge",
+            "hints": {
+                "allow": [
+                    "POST"
+                ]
+            }
+        }
+    }
+}
+```
+
+## Add a phone
+
+<ApiOperation method="post" url="/idp/myaccount/phones" />
+
+Create an UNVERIFIED status phone for either `SMS` or `CALL` method to the user's MyAccount setting.
+
+### Required scope and role
+
+An Okta scope of `okta.myAccount.phone.manage` is required to use this endpoint.
+
+> **Note:** Admin users aren't allowed to call the `/idp/myaccount/phones` POST endpoint.
+
+#### Request path parameters
+
+N/A
+
+#### Request body
+
+This API requires a [My Phone Request object](#my-phone-request-object) as its request body.
+
+```bash
+{
+  "profile": {
+      "phoneNumber" : "+15555555555"
+  },
+   "sendCode": true,
+   "method": "SMS"
+}
+```
+
+### Response body
+
+The requested [My Phone object](#my-phone-object)
+
+### Error Responses
+
+If an invalid phone number is passed to `profile` in the request body, the response returns a 400 BAD REQUEST with error code E0000001.
+
+If an invalid `method` is passed in the request body, the response returns a 400 BAD REQUEST with error code E0000001.
+
+If the phone authenticator isn't enabled for `method` on the org, the response returns a 403 FORBIDDEN with error code E0000038.
+
+If the number of phone factors for the current user already reaches the maximum allowed per user or the phone factor is failed to create, the response returns a 500 INTERNAL SERVER ERROR with error code E0000009.
```

Comment:
this shouldn't be the case, we should never return a 500. If this is the case on the API then it is a bug and we should fix it.

Reply:
@zeusandrade-okta that bit of content is from the reference doc.

Reply:
Fixed, as per our Slack discussion.